migratedb.d/04.sql

```diff
@@ -101,7 +101,8 @@ BEGIN
     );
 
     INSERT INTO file_events(id,title,description)
-    VALUES (10, 'uploaded'    , 'Uploaded to the S3 inbox'),
+    VALUES ( 5, 'registered'  , 'Upload to the inbox has started'),
+           (10, 'uploaded'    , 'Uploaded to the S3 inbox'),
```

Comment:
is this specific to the s3 inbox or to the inbox ?

Reply:
we use the s3 inbox, but it could be used by other services :) but I can see that this should probably be the same in both 'registered' and 'uploaded' :)

Reply:
i, see. would be nice to make it generic not s3 specific, unless that is actually the case

Reply:
```suggestion
           (10, 'uploaded'    , 'Upload to the inbox has finished'),
```